### ISSUE TITLE
Bulk export of playlists

### ISSUE THREAD

**Issue by Merkwurdichliebe** [navidrome/navidrome#2163]
### Is your feature request related to a problem? Please describe.

Relying on navidrome as a main audio player makes the playlists in the database very valuable and imho there should be a way to back them up into `m3u8` files without making a full copy of the database. Currently it's only possible to export playlists one at a time.

### Describe the solution you'd like

A button labeled "export" next to the existing "make public", "make private" and "delete" would be nice.

### Describe alternative solutions that would also satisfy this problem

"Export all playlists" somewhere in the user menu would be fine too.

Thanks for all the great work.

**Comment by deluan** [comment]
I think you should do a full copy of the database, as it also contains important information like ratings, favourites, play counts.

Anyway, while the functionality is not implemented, here's a script you can use to export all playlists:

```
curl -s "https://your-server,com/rest/getPlaylists?u=<USERNAME>&p=<PASSWORD>&c=my-script&f=json&v=1.15.0"  jq -r '.["subsonic-response"].playlists.playlist[] | .id + " \"" + .name + "\""' | while IFS= read -r id name; do
   navidrome pls -p $id -o ${name}.m3u
done
```


You may need to pass `-c` to the `navidrome pls` command, to point to your configuration file.

**Comment by Merkwurdichliebe** [comment]
Thank a lot for replying. I believe you're missing a pipe character before `jq`.

Anyway, it doesn't work for me. I needed to add `./` so that it find `navidrome` in the current dir, otherwise I get `Command not found`. When I do that, it runs `navidrome` on each iteration correctly, but I get `FATAL Playlist not found, name=` and then the playlist `id` followed by the escaped name, like this `\"My Playlist\"`.

This happens whether I point to my `.toml` file or not.

I've added `echo $id $name` to check the variables and they are assigned correctly on each iteration, so I don't know what's wrong here.

BUT, this is a very special-use case and I don't want to tie you up with this. I imagine the feature will be implemented someday and in the meantime I'll just backup the database (which I already do anyway, I just wanted to know how to get the playlists out of there in case I needed them).

Thanks a lot for your time.
